app/controllers/cuisines_controller.rb

```diff
@@ -5,12 +5,6 @@ class CuisinesController < ApplicationController
   # GET /cuisines.json
   def index
     render json: Cuisine.all.order(:name)
```

Comment:
You can add this in the model under `scope` so the default will be ordered by name